assets/js/blocks/attribute-filter/editor.scss

```diff
@@ -11,18 +11,13 @@
 	}
 	.components-placeholder__fieldset {
 		display: block; /* Disable flex box */
-
-		p {
-			@include font-size(14px);
```

Comment:
Regarding this font size and the one below, they were making the placeholder text and button smaller.

_How it looked:_
![Screenshot from 2020-05-21 15-58-43](https://user-images.githubusercontent.com/3616980/82652144-6aeb1f80-9c1d-11ea-868e-6043b281afab.png)

_How it looks after removing those declarations:_
![Screenshot from 2020-05-21 15-58-13](https://user-images.githubusercontent.com/3616980/82652185-78a0a500-9c1d-11ea-8b0e-2c75724fdef0.png)

I took a look at the designs, they use the old placeholder style, but I don't see a reason why that text should be smaller than the rest:
![imatge](https://user-images.githubusercontent.com/3616980/82652238-940bb000-9c1d-11ea-939e-ef25c3496563.png)